src/assets/scss/material.orcid.overwrites.scss/button.scss

```diff
@@ -21,10 +21,41 @@ button.mat-button-wrap-text-with-icon {
   border: solid 1px rgba(0, 0, 0, 0.12);
   margin-top: 12px;
 
-  mat-icon {
+  a {
     position: absolute;
-    left: 5%;
-    top: 20%;
+    left: 20px;
+    top: 5px;
+    img {
+      width: 24px;
+    }
+  }
+
+  .columns-4 & {
+    span {
+      display: inline-block;
+      width: 180px;
+      margin: auto;
+      word-break: break-word;
+      white-space: normal;
+      line-height: 24px;
+      min-height: 36px;
+    }
+  }
+}
+
+[dir='rtl'] button.mat-button-wrap-text-with-icon {
+  width: 100%;
+  border: solid 1px rgba(0, 0, 0, 0.12);
+  margin-top: 12px;
+
+  a {
+    position: absolute;
+    right: 20px;
+    left: unset;
+    top: 5px;
+    img {
+      width: 24px;
```

Comment:
RTL is on top of the non-RTL CSS definition, so there is no need to repeat what stays the same, only the attributes that are overwritten. 

Meaning that you can remove all the attributes that are the same like:

```
width: 100%;
position: absolute;
top: 5px;
img {
      width: 24px;
```

normally only `left` and `right` related attributes are overwritten

Reply:
Ok ok. I will remote the other attributes